plugins/services/src/js/service-configuration/ServiceConfigDisplay.js

```diff
@@ -4,7 +4,7 @@ import { MountService } from "foundation-ui";
 
 import ErrorsAlert from "#SRC/js/components/ErrorsAlert";
 import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
-import { translateErrorMessages } from "#SRC/js/utils/ErrorMessageUtil";
+import ErrorMessageUtil from "#SRC/js/utils/ErrorMessageUtil";
```

Comment:
Why was this changed?

Reply:
For some reason, Typescript doesn't like destructured imports. It errors with "`translateErrorMessages` is not a function" when I do a destructured import